packages/mobile/src/screens/settings-screen/AppearanceSettingsRow.tsx

```diff
@@ -14,6 +15,8 @@ import { SettingsRow } from './SettingsRow'
 import { SettingsRowContent } from './SettingsRowContent'
 import { SettingsRowDescription } from './SettingsRowDescription'
 
+const isStaging = Config.ENVIRONMENT === 'staging'
```

Comment:
Maybe instead we can check for `Config.ENVIRONMENT !=== 'production'` or something?